sycl/ReleaseNotes.md

```diff
@@ -1,3 +1,370 @@
+- Ensured that compiler-generated integration header/footer are warning-free to avoid breaking `-Werror` builds, especially when 3rd-party host compiler is used. intel/llvm#15171 intel/llvm#15175 intel/llvm#15197
+- SYCL `joint_matrix` extension basic functionality is now built on top of `SPV_KHR_cooperative_matrix` extension. intel/llvm#16045 intel/llvm#15038
+- Added more supported aspects for CPU AOT target. intel/llvm#15263
+- Added diagnostic for bad argument with `-fsycl-device-obj`. intel/llvm#15381
+- Added a warning for cases when kernel-only attributes are applied to non-kernel functions. intel/llvm#15154
+- Fixed misleading diagnostic about non-external function/variable when applying attributes like  `[[sycl_device]]` or `[[intel::device_indirectly_callable]]` on  functions/variables without external linkage. intel/llvm#15372
+- The `-fsycl-link=image` behavior is updated to package host objects similarly to `-fsycl-link=early`, ensuring proper linking, especially on Windows. intel/llvm#15539
+- Enabled generation of approximate `div`/`sqrt` with `-ffast-math` for NVTPX. intel/llvm#15553
+- Added extra optimization passes in Native CPU pipeline. intel/llvm#14380
+- Updated implementation of `-fsycl-host-compiler` option to only use user-provided hints (i.e. `PATH`) to locate the specified compiler. Previously this option also took into account a few extra locations implicitly known to the toolchain, potentially causing incorrect binary to be used. intel/llvm#15769
+- Deprecated `[[intel::reqd_sub_group_size]]`, the official SYCL 2020 spelling should be used instead (with `sycl::` namespace). intel/llvm#15798
+- Disabled ITT annotations in device code by default to reduce device code size. intel/llvm#14910
+- Enabled floating point atomics via `atomicrmw` instructions for Native CPU. intel/llvm#15888
+- Enabled nonsemantic debug info by default. This should slightly improve debugging experience. intel/llvm#16120
+
+### SYCL Library
+
+- Enabled check for use of `SYCL_PI_TRACE` on all Linux to inform users that `SYCL_UR_TRACE` should be used instead. intel/llvm#15113
+- Improved GDB printers for SYCL types/values. intel/llvm#15076
+- Renamed `ur` to `ur.call` in XPTI traces. intel/llvm#14971
+- Refactored XPTI framework to use 128-bit keys for collision elimination and added support for 64-bit universal IDs for backward compatibility. Previous 64-bit hash values were used to represent trace points and this has led to a few of hash  collisions. intel/llvm#14467
+- Made calling `command_graph::begin_recording` repeatedly an error. intel/llvm#15192
+- Reduced number of devicelib files that have to be redistributed for CUDA and HIP backends. As a side effect, device libraries cannot be selectively used anymore either all of them are linked, or none. intel/llvm#15048 intel/llvm#15055
```

Comment:
More importantly the second PR intel/llvm#15055 added devicelib support for AMD, which enable uses of `std::` math built-in. We could add a line like this:

> - Added some C++ standard library math function support on AMD. intel/llvm#15055

Reply:
Ok, thanks, fixed per suggestion.